opendcs-rest-api/src/main/java/org/opendcs/odcsapi/util/ApiBasicClient.java

```diff
@@ -143,30 +145,27 @@ public void disconnect( )
 				if (input != null)
 					input.close();
 			}
-			catch (Exception e)
+			catch (IOException ex)
 			{
-				System.out.println("Error closing input: "+e.getMessage());
-				e.printStackTrace();
+				throw new IOException("Error closing input: ", ex);
 			}
-			try
+            try
 			{
-				if (output != null)
-					output.close();
-			}
-			catch (Exception e)
+                if (output != null)
+                    output.close();
+            }
+			catch (IOException ex)
 			{
-				System.out.println("Error closing ouput: "+e.getMessage());
-				e.printStackTrace();
-			}
+				throw new IOException("Error closing output: ", ex);
+            }
 			try
 			{
 				if (socket != null)
 					socket.close();
 			}
-			catch (Exception e)
+			catch (IOException ex)
 			{
-				System.out.println("Error closing socket: "+e.getMessage());
-				e.printStackTrace();
+				throw new IOException( "Error closing socket: ", ex);
```

Comment:
This doesn't add much context, I'd just remove the catch since it's throwing the same exception anyways